verifiers/internal/gha/rekor.go

```diff
@@ -67,13 +67,17 @@ func verifyTlogEntryByUUID(ctx context.Context, rekorClient *client.Rekor, entry
 			return nil, errors.New("expected matching UUID")
 		}
 		// Validate the entry response.
-		return verifyTlogEntry(ctx, rekorClient, entry)
+		return verifyTlogEntry(ctx, entry, true)
```

Comment:
iiuc, we'll perform an online verification by default to be consistent with our existing code. And we'll provide an option to turn it off in the futuer, e.g. `rekor-log-offline`?

Reply:
Right now the existing code still validates the inclusion proof (which requires online).

I haven't configured augmenting the offline verification with an online proof yet, this code just checks verification of the offline material. I'll file an issue though to augment with an option to check inclusion online as well.